Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -192,77 +231,68 @@ void updateLayersForInteractionRegions(const RemoteLayerTreeNode& node)
     NSUInteger insertionPoint = 0;
     for (const WebCore::InteractionRegion& region : node.eventRegion().interactionRegions()) {
```

Comment:
Things were getting out of hand here. @mwyrzykowski already had to fix some hidden mistake in the last patch and I need to _add more logic_ for reuse. So there's a bit of a refactor, hopefully for the best!
The diff is not great but it's much more readable now if you want to just do "View file" and look at the whole thing :)